proto/task_service.proto

```diff
@@ -35,6 +35,7 @@ message CreateTaskRequest {
   batch_plan.TaskId task_id = 1;
   batch_plan.PlanFragment plan = 2;
   uint64 epoch = 3;
+  common.VNodeRanges vnode_ranges = 4;
```

Comment:
Should it be a list? like 

```
repeated common.VNodeRanges vnode_ranges = 4;
```

Reply:
`VNodeRanges` is multiple ranges
```
// Records the vnodes owned by a parallel unit.
//
// `starts`: [s1, s2, ...], `ends`: [e1, e2, ...] means ranges [s1, e1], [s2, e2], ... (both ends included)
message VNodeRanges {
  repeated uint64 starts = 1;
  repeated uint64 ends = 2;
}
```

It can be sth like `repeated VNodeRange` or `VNodeRanges`. I just drafted it without too much consideration 🥸

Reply:
+1 for `repeated Range`, then `zip_`**`eq`** is implied.